README.md

```diff
@@ -53,6 +53,15 @@ Latest installers with PyPy 3.9 in the base environment:
 | OS X    | x86_64                | [Miniforge-pypy3-MacOSX-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge-pypy3-MacOSX-x86_64.sh) |
 | Windows | x86_64                | [Miniforge-pypy3-Windows-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge-pypy3-Windows-x86_64.exe) |
 
+<details><summary>Mambaforge (Deprecated as of September 2023)</summary>
+
+In [#277](https://github.com/conda-forge/miniforge/pull/277), the functionality
+of Mambaforge was incorporated into Miniforge. We have however kept the
+download links of Mambaforge active to help those that have automated
+workflows. Because of this, as of September 2023, the new usage of Mambaforge
+is discouraged. Bug reports specific to Mambaforge will be closed as
+`wont-fix`.
```

Comment:
Maybe that is missing here is something along the lines that the next mambaforge and miniforge releases will be virtually the same with the exception of shipping mamba in one vs the other, right?

Reply:
`mamba` is in both now.

Reply:
I may be wrong but I don't get that idea from this message. Maybe we should just say that above.

Reply:
The idea is that since both are equivalent, we are going to predominantly suggest using Miniforge and just keep Mambaforge around for backwards compatibility. Hence, if you need to start using our installers today, we are saying that you should go for Miniforge.

Reply:
Yep. That much us clear but functionality is different from identical. I may be exaggerating but instead of "the functionality of Mambaforge was incorporated into Miniforge." maybe something along the lines:

"as of release x.x.x Mambaforge and Miniforge are identical and we recommend the use of Miniforge b/c Mambaforge is just a link to it with the goal of preventing link rot."

Reply:
I guess to me they were identical since they install to different default directories. I can make this more clear.

Reply:
Oh. Yeah, to me too. I'm trying to read that as a user that is not 100% in tune with conda/conda-forge events and decisions.

Reply:
I finally sat down and read @jaimergp 's message on https://github.com/conda-forge/conda-forge.github.io/pull/1995 

I pulled most of his language.